src/Stratis.Bitcoin.IntegrationTests/Wallet/SendingToAndFromManyAddressesSpecification_Steps.cs

```diff
@@ -56,12 +56,15 @@ private void two_connected_nodes()
                     .Connect(NodeOne, NodeTwo)
                     .AndNoMoreConnections()
                 .Build();
+
+            this.CoinBaseMaturity = 1;
```

Comment:
Here you can do `KnownNetworks.RegTest.Consensus.CoinbaseMaturity = 1;` in the beforeTests method.

Reply:
Done